azure-functions-powershell-library/BundledBindings/ActivityTriggerBinding.cs

```diff
@@ -0,0 +1,28 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using AzureFunctions.PowerShell.SDK.Common;
+using Common;
+using Microsoft.Azure.Functions.PowerShellWorker;
+using System.Management.Automation.Language;
+
+namespace AzureFunctions.PowerShell.SDK.BundledBindings
+{
+    public class ActivityTriggerBinding : IInputBinding
+    {
+        public override string BindingAttributeName => Constants.AttributeNames.ActivityTrigger;
+
+        public override string BindingType => Constants.BindingNames.ActivityTrigger;
+
+        public override BindingInformation ExtractBinding(AttributeAst attribute, ParameterAst parameter)
+        {
+            BindingInformation bindingInformation = new BindingInformation();
+            bindingInformation.Name = parameter.Name.VariablePath.UserPath;
+            bindingInformation.Direction = (int)BindingDirection;
```

Comment:
This could be modified to the below if we change the `Direction` member of `BindingInformation` to be of the type `Directions`.
```suggestion
            bindingInformation.Direction = BindingDirection;
```

Reply:
Done